airflow/models/dag.py

```diff
@@ -227,7 +227,7 @@ def __init__(
         self._description = description
         # set file location to caller source path
         self.fileloc = sys._getframe().f_back.f_code.co_filename
-        self.task_dict = dict()  # type: Dict[str, TaskInstance]
+        self.task_dict = dict()  # type: Dict[str, BaseOperator]
```

Comment:
Is this correct?

Reply:
Yes :( We actually fill it with Operators, not TaskInstances. The mixing of "operator" and "task" throughout the codebase is rather confusing.

Reply:
I changed it in this PR because my IDE (PyCharm) was not suggesting the correct methods.